pkg/gds/members.go

```diff
@@ -270,11 +249,21 @@ func (s *Members) Summary(ctx context.Context, in *api.SummaryRequest) (out *api
 		since = time.Now().Add(-time.Hour * 24 * 30)
 	}
 
-	// TODO: Add paramater to fetch details for a specific VASP
-
 	// Create response
 	out = &api.SummaryReply{}
 
+	if in.Vasp != "" {
```

Comment:
Nice - so supplying the `Vasp` is completely optional and if it's omitted from the request then the `Vasp` in the response is just `nil` - nice one! I haven't looked at the tests yet, but we'll want an explicit test for this since the `BFF` depends on this behavior (to prevent regressions)

Reply:
Thanks, I'll add a check to the test to assert that specifically.